teku/src/main/java/tech/pegasys/teku/cli/subcommand/VoluntaryExitCommand.java

```diff
@@ -104,6 +108,26 @@ public class VoluntaryExitCommand implements Runnable {
       arity = "1")
   private boolean confirmationEnabled = true;
 
+  @CommandLine.Mixin(name = "Data")
+  private ValidatorClientDataOptions dataOptions;
+
+  @CommandLine.Option(
+      names = {"--public-keys-filter"},
+      description = "Filter the exiting validators by public keys.",
+      paramLabel = "<STRINGS>",
+      split = ",",
+      arity = "0..1")
+  private List<String> pubKeyFilter;
+
+  @CommandLine.Option(
+      names = {"--skip-keymanager-keys"},
```

Comment:
Thinking twice lets put it in the positive form by having `--include-keymanager-keys` which is true by default
and having a description like "Include validator keys managed via keymanager APIs"

Reply:
Ok for the inversion but I think in that case it should be false by default to handle the case when the user doesn't specify a `--data-validator-path` (which has a default value) and only use the `--validator-keys` option. In that case the "key manager" validators will be exited without the user asking for it

Reply:
ah yes --skip-keymanager-keys has true by default so --include-keymanager-keys false by default